extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -389,8 +389,14 @@ public boolean isAnyPropertySet() {
         @ConfigItem(defaultValueDocumentation = "depends on dialect")
         public Optional<Boolean> jdbcWarnings;
 
+        /**
+         * If set, Hibernate will log queries that took more than specified number of milliseconds to execute.
+         */
+        @ConfigItem
+        public Optional<Long> queriesSlowerThanMs;
```

Comment:
Why don't you make it a Duration instead? That would be more inline with what we usually do.

Reply:
That's what I tried initially. But I didn't see a way duration can handle milliseconds.